litmus-portal/authentication/api/handlers/rest/project_handler.go

```diff
@@ -324,12 +335,12 @@ func SendInvitation(service services.ApplicationService) gin.HandlerFunc {
 			return
 		}
 		if member.Role == nil {
-			c.JSON(400, gin.H{"message": "Enter a vaild role"})
+			c.JSON(utils.ErrorStatusCodes[utils.ErrInvalidRole], presenter.CreateErrorResponse(utils.ErrInvalidRole))
 			return
 		}
 
 		if *member.Role != entities.RoleEditor && *member.Role != entities.RoleViewer {
-			c.JSON(utils.ErrorStatusCodes[utils.ErrInvalidRequest], presenter.CreateErrorResponse(utils.ErrInvalidRequest))
+			c.JSON(utils.ErrorStatusCodes[utils.ErrInvalidRole], presenter.CreateErrorResponse(utils.ErrInvalidRole))
 			return
 		}
 
```

Comment:
Code block from 337 to 345 can be replaced with this:
```suggestion
			// Validating member role
			if mem.Role == nil || ( *mem.Role != entities.RoleProjectEditor && *mem.Role != entities.RoleProjectViewer) {
				c.JSON(utils.ErrorStatusCodes[utils.ErrInvalidRole], presenter.CreateErrorResponse(utils.ErrInvalidRole))
				return
			}
```